ophyd/ophydobj.py

```diff
@@ -33,11 +33,11 @@ class OphydObject:
 
     _default_sub = None
 
-    def __init__(self, *, name=None, parent=None, device_labels=None):
+    def __init__(self, *, name=None, parent=None, labels=None):
```

Comment:
The changes here are opposite than the statement in the commit title. I'm a bit confused - do we eventually prefer `labels` over `device_labels`?

Reply:
oops yes wrote too fast, went from `device_labels` to `lablels` will edit commit
thanks!